### ISSUE TITLE
refactor: line-height shouldn't wear unit

### ISSUE THREAD

**Issue by crotru** [Decathlon/vitamin-web#29]
# Bug Report

`line-height` cascades down and shouldn't wear unit

**Current behavior**

```css
element {
  font-size: 2rem;
  line-height: 3rem;
}
````
It implies that any children of `element` without explicit `line-height` definition will have a 3rem line-height whatever its font-size may be.

**Expected behavior**

`line-height` should be a multiplicator of font-size. 
```css
element {
  font-size: 2rem;
  line-height: 1.5;
}
````
That way `element` children will have a consistent `line-height` whatever `element` font-size value or unit is.

**Related code**

in /packages/css/src/design-tokens/typography.css

![image](https://user-images.githubusercontent.com/22882255/103233909-91550400-493e-11eb-96e8-c1520a75d85f.png)

**Comment by lauthieb** [comment]
Oh nice you're right, thanks!!
I will check with our designers what could be the best percentage to apply > 1 for our different line heights.

**Comment by lauthieb** [comment]
Update, here are the new values available there: https://www.decathlon.design/726f8c765/p/860e14-typography/b/208916 ![image](https://user-images.githubusercontent.com/9600228/104650120-043bda00-56b6-11eb-886e-0c3387e1a317.png)

**Comment by crotru** [comment]
Hi there.
I realize there were a confusion between this present issue (about removing units) and [this one](#30) (about changing their values).
This one is not a designer issue but a tech one

I re-open :) 

---

A bit more precision on the reason for this :

Cascade can break children of an element holding an "united-line-height" : 
https://codepen.io/__sch__/pen/MWbQpqo?editors=1100

Line-height should only be multiplicators : the declaration `line-height: 1.5` induces that proper physical line height will be : your font-size * 1.5

**Comment by lauthieb** [comment]
Thanks @crotru for repoting this!
That was because Tailwind uses `rem` per default for their font sizes definition. Here is the Pull Request to fix this: #180
